packages/template-chakra-storefront/src/pages/product-list/partials/color-refinements.jsx

```diff
@@ -32,15 +32,26 @@ const ColorRefinements = ({filter, toggleFilter, selectedFilters}) => {
 
                 const styles = recipe({variant: 'circle', selected: isSelected})
 
+                const messages = {
+                    ariaLabel: intl.formatMessage(
```

Comment:
Let's make the name clearer to make it possible to add a new ariaLabel down the road. e.g adding a new functional Button will require different ariaLabel
```suggestion
                    colotHitariaLabel: intl.formatMessage(
```

Reply:
Please double check this for the rest of the code